R/actions.R

```diff
@@ -39,10 +39,10 @@ use_tidy_actions <- function() {
   new <- use_action_check_full() && use_action_pr_commands()
 
   if (!new) {
-    return(invisible(FALSE))
+    return(invisible(new))
   }
 
-  invisible(TRUE)
+  invisible(new)
 }
```

Comment:
Sorry 😂 I meant: can't this be replaced with a simple `invisible(new)`? It covers the `TRUE` and `FALSE` case and we do it elsewhere when there's no additional work done for one case.

Reply:
Oh right, I did this now